mono/metadata/boehm-gc.c

```diff
@@ -34,6 +34,11 @@
 #include <mono/utils/mono-os-mutex.h>
 #include <mono/utils/mono-counters.h>
 
+#ifdef _MSC_VER
+	// Quiet Visual Studio linker warning, LNK4221, in cases when this source file intentional ends up empty.
+	void __mono_win32_boehm_gc_quiet_lnk4221(void) {}
```

Comment:
Maybe these could be ignored by adding ` /ignore:4221` in `Properties->Linker->Command Line` instead of adding these fake declarations in the source?